zoneminder.json

```diff
@@ -0,0 +1,45 @@
+{
+    "ZoneMinder": {
+        "containers": {
+            "zoneminder": {
+                "image": "aptalca/zoneminder-1.29",
+                "launch_order": 1,
+                "opts": [
+                         [
+                          "--privileged=true",
+                          ""
+                          ]
+                         ],
+                "ports": {
+                    "80": {
+                        "description": "Port for running ZoneMinder. You shouldn't open this externally (it's unsecure). Do not use 80 (it will interfere with RockStor)",
+                        "host_default": 8080,
```

Comment:
Thanks for this note, but happy to say that as of 3.8-14.02, Rockstor service(via nginx) no longer uses port 80. Let's leave this note here, however till we release 3.8-15.